packages/core/tests/integration/components/navi-cell-renderers/time-dimension-test.js

```diff
@@ -0,0 +1,161 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const TEMPLATE = hbs`
+  <NaviCellRenderers::TimeDimension
+    @data={{this.data}}
+    @column={{this.column}}
+    @request={{this.request}}
+  />`;
+
+const data = {
+  dateTime: '2016-06-03 11:12:13.000',
+  'os|id': 'All Other',
+  'os|desc': 'All Other',
+  uniqueIdentifier: 172933788,
+  totalPageViews: 3669828357
+};
+
+const column = {
+  type: 'time-dimension',
+  displayName: 'Date',
+  attributes: {
+    name: 'dateTime',
+    parameters: {
+      grain: 'day'
+    }
+  }
+};
+
+const request = {
```

Comment:
Could this file be `.ts`? Not sure what shape this request is because requestV2 doesn't have a timeGrain property

Reply:
Good point, this is actually a mock of a requestv2 fragment